pyproject.toml

```diff
@@ -15,9 +15,14 @@ profile = "black"
 
 [tool.mypy]
 files = "src"
-python_version = "3.6"
+exclude = [
+    'decaylanguage\.modeling\.*',
+]
+python_version = "3.9"
 warn_unused_configs = true
+warn_unused_ignores = true
 check_untyped_defs = true
+strict = true
```

Comment:
I would not recommend starting with `strict=True` - that's great eventually, but it's probably better to go in stages. Here's the current list (in flag form, because that's what you get in `mypy --help`): `--warn-unused-configs, --disallow-any-generics, --disallow-subclassing-any,
                            --disallow-untyped-calls, --disallow-untyped-defs, --disallow-incomplete-defs, --check-untyped-defs, --disallow-untyped-
                            decorators, --no-implicit-optional, --warn-redundant-casts, --warn-unused-ignores, --warn-return-any, --no-implicit-
                            reexport, --strict-equality`.

You can also apply strict (or these finer-grained settings) to just one submodule at a time, I bet that's what you actually want to do.

But if it's already nearly passing strict, great! :)

Reply:
Yeah, I've been working in batches but it's taken me time to read about  type checking and then implement it in the tens and tens of places. Not too far off a draft v0, actually.